e2e/test/Helpers/CustomWebProxy.cs

```diff
@@ -9,12 +9,12 @@ namespace Microsoft.Azure.Devices.E2ETests.Helpers
 {
     public class CustomWebProxy : IWebProxy
     {
-        private readonly TestLogger s_testLog;
+        private readonly MsTestLogger _logger;
```

Comment:
Any reason this needs to be the child class instance, rather than the parent class `TestLogger`? Isn't the `.Trace()` API call available on the parent class directly?